test/e2e/suite_test.go

```diff
@@ -1645,3 +1646,25 @@ func checkMetricsFiltersPresence(t *testing.T, ctx context.Context, f *framework
 	}
 	return nil
 }
+
+func checkInjectedContainersPresence(t *testing.T, ctx context.Context, f *framework.E2eFramework, dcKey framework.ClusterKey) error {
```

Comment:
It doesn't hurt to have it, but do we need these checks in e2e tests? Are the integration and unit tests sufficient?

Reply:
integration and unit tests don't involve cass-operator. I like to make sure the new behaviors are checked in the e2e tests. It has proven useful with my last PR 😅

Reply:
>integration and unit tests don't involve cass-operator 

fair point :)